orga/tests/integration/components/routes/authenticated/sup-students/list-items-test.js

```diff
@@ -0,0 +1,102 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, fillIn } from '@ember/test-helpers';
+import sinon from 'sinon';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | routes/authenticated/sup-students | list-items', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('noop', sinon.stub());
+  });
+
+  test('it should show title of team page', async function(assert) {
+    // when
+    await render(hbs`<Routes::Authenticated::SupStudents::ListItems @triggerFiltering={{noop}}/>`);
+
+    // then
+    assert.contains('Étudiants');
+  });
+
+  test('it should display the header labels', async function(assert) {
+    // given
+    this.set('students', []);
+
+    // when
+    await render(hbs`<Routes::Authenticated::SupStudents::ListItems @students={{students}} @triggerFiltering={{noop}}/>`);
+
+    // then
+    assert.contains('Numéro étudiant');
+    assert.contains('Nom');
+    assert.contains('Prénom');
+    assert.contains('Date de naissance');
+  });
+
+  test('it should display a list of students', async function(assert) {
+    // given
+    const students = [
+      { lastName: 'La Terreur', firstName: 'Gigi', birthdate: new Date('2010-02-01') },
+      { lastName: 'L\'asticot', firstName: 'Gogo', birthdate: new Date('2010-05-10') },
+    ];
+
+    this.set('students', students);
+
+    // when
+    await render(hbs`<Routes::Authenticated::SupStudents::ListItems @students={{students}} @triggerFiltering={{noop}}/>`);
+
+    // then
+    assert.dom('[aria-label="Étudiant"]').exists({ count: 2 });
+  });
+
+  test('it should display the firstName, lastName and birthdate of student', async function(assert) {
+    // given
+    const students = [{ lastName: 'La Terreur', firstName: 'Gigi', birthdate: new Date('2010-02-01') },];
+
+    this.set('students', students);
+
+    // when
+    await render(hbs`<Routes::Authenticated::SupStudents::ListItems @students={{students}} @triggerFiltering={{noop}}/>`);
+
+    // then
+    assert.contains('La Terreur');
+    assert.contains('Gigi');
+    assert.contains('01/02/2010');
+  });
+
+  module('when user is filtering some users', function() {
+    test('it should trigger filtering with lastname', async function(assert) {
```

Comment:
Les `it`  manquent de context  `when user is filtering some users => it should trigger filtering with lastname`
C'est quand on chercher un étudiant par son nom => `it filters with lastname`

Reply:
Même point pour `it should trigger filtering with firstname`

Reply:
j'ai pas compris 😅

Reply:
quand je lit le contexte du test : 
`when user is filtering some users it should trigger filtering with lastname`  c'est vague comme description je pense qu'on peut avoir de meilleurs "commentaires" qui soit plus descriptifs de la réalité.
`when user is filtering some users` c'est vague
Quelque chose comme `when the user fill the last name filter` => `it filters students by lastname`

Reply:
le contexte utilisé ici est volontairement générique pour mettre à l'intérieur tous les filtres, sinon on met un contexte pat test et donc le contexte ne sert plus à grand chose, autant tout mettre dans le test. Après on peut changer la formulation.